src/server/services/helpers/readFileIndex.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function readFileIndex (indexData) {
+    let indexObject = {};
+    try {
+        indexObject = JSON.parse(indexData)
+    } catch (error) {
+        console.error("Sorry, i can`t parse to JSON indexDATA file. \n" + error);
+    }
+
+    return indexObject.currentIndex;
```

Comment:
if the error part is triggered than `indexObject` is `{}` and you return `currentIndex` from an empty object?

Reply:
Yep. Bottle neck?